DllMain.cpp

```diff
@@ -0,0 +1,18 @@
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include <HFL/Source/HFL.h>
+
+BOOL APIENTRY DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved)
+{
+	if (fdwReason == DLL_PROCESS_ATTACH)
+	{
+		HFLInit();
+		DisableThreadLibraryCalls(hinstDLL);
```

Comment:
This call seems standard in Outpost2 missions, so I left it in. Not sure if it is necessary.

Reply:
It's not required, but from what I understand, it can give a slight performance boost, as `DllMain` doesn't need to be called for this module for each thread the game creates.

Reply:
Ok, thanks. I had read about the performance boost.